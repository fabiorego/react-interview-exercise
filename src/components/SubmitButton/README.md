Submit button:

```jsx
<SubmitButton>Submit Form</SubmitButton>
```
