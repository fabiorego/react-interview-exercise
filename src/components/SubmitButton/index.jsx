import React from "react";

const SubmitButton = ({ children }) => {
  return (
    <button className="btn btn-lg btn-primary btn-block" type="submit">
      {children}
    </button>
  );
};

export default SubmitButton;
