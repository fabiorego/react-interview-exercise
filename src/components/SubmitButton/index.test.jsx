import React from "react";
import SubmitButton from "./index";
import renderer from "react-test-renderer";

test("SubmitButton snapshot", () => {
  const component = renderer.create(<SubmitButton>Submit Button</SubmitButton>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
