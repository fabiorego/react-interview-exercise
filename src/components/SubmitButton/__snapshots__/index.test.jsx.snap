// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitButton snapshot 1`] = `
<button
  className="btn btn-lg btn-primary btn-block"
  type="submit"
>
  Submit Button
</button>
`;
